server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -506,7 +506,7 @@ export class ScribeLambda implements IPartitionLambda {
             },
             (error) => {
                 this.context.error(error, {
-                    restart: true,
+                    restart: false,
```

Comment:
>restart: false, [](http://example.com/codeflow?start=20&length=15)

What does this do? and why do we change it?

Reply:
We use this to restart scribe when the checkpointing process throws an error.  I noticed that all the errors that we are restarting are due to errors in writing to mongodb. Given that checkpointing is not in the critical path and that restarting scribe is a very heavy operation and most of these errors will not be helped by scribe restarts, I thought it might be useful to do this. If you think this needs more discussion, I happy to revert it back. It is not directly related to this PR.

Reply:
Thanks for explaining. The change makes sense.